src/flint/fmpz.jl

```diff
@@ -1267,6 +1267,37 @@ function divisible(x::fmpz, y::Int)
               (Ref{fmpz}, Int), x, y))
 end
 
+@doc Markdown.doc"""
+    divisors(a::fmpz)
+
+Return the positive divisors of $a$ in an array, not necessarily in growing
+order. We require $a \neq 0$.
+"""
+function divisors(a::fmpz)
+   iszero(a) && throw(DomainError("Argument must be non-zero"))
+
+   divs = fmpz[one(FlintZZ)]
+   isone(a) && return divs
+
+   for (p,e) in factor(a)
+      ndivs = deepcopy(divs)
```

Comment:
```suggestion
      ndivs = copy(divs)
```
Makes it a bit faster.